source/docs/articles/sites/security/SSL-TLS.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Pantheon SSL/TLS support 
+description: Pantheon's recommendation regarding SSL/TLS in light of PCI DSS v3.1.
+category:
+  - security
+  - developing
+  - managing
+
+---
+We recommend organizations that are working to achieve PCI compliance while using Pantheon services engage a third-party service provider like CloudFlare to terminate SSL/TLS connections.  Terminating SSL/TLS connections with a service provider that specializes in providing SSL/TLS gives you greater control over your configuration and ensures your service will track closely with changes to PCI DSS.
```

Comment:
This needs an introduction that disambiguates/defines three key acronyms:

- HTTPS
- SSL
- TLS

Many developers conflate all three. We should clarify this up-front so that people can become more educated, and so that they do not misread statements as "we have ended support for all versions of SSL" to mean we are dropping HTTPS support from the platform.